lib/ibmdb.js

```diff
@@ -26,15 +26,7 @@ var Transaction = IBMDB.Transaction = SQLConnector.Transaction;
 exports.initialize = function(ds, cb) {
   ds.connector = new IBMDB('IBMDB', ds.settings);
   ds.connector.dataSource = ds;
-  if (cb) {
-    if (ds.settings.lazyConnect) {
-      process.nextTick(function() {
-        cb();
-      });
-    } else {
-      ds.connector.connect(cb);
-    }
-  }
```

Comment:
You may consider "console.warn('ibm connectors natively support lazy connect via ... see link")"